### PR TITLE
Update master to original repo

### PR DESCRIPTION
**Pull request by tobiasweibel**

### CODE REVIEW

File: AFOAuth1Client/AFOAuth1Client.m [hunk 1]
```diff
@@ -317,9 +326,20 @@ - (void)authorizeUsingOAuthWithRequestTokenPath:(NSString *)requestTokenPath
 
         NSMutableDictionary *parameters = [NSMutableDictionary dictionary];
         parameters[@"oauth_token"] = requestToken.key;
+
         NSMutableURLRequest *request = [super requestWithMethod:@"GET" path:userAuthorizationPath parameters:parameters];
         [request setHTTPShouldHandleCookies:NO];
-        self.onServiceProviderRequest(request);
```
Comment (tobiasweibel) [inline AFOAuth1Client/AFOAuth1Client.m]: onServiceProviderRequest is replaced by serviceProviderRequestHandler

**Comment by tobiasweibel** [comment]
@larsxschneider Can you please review it? I will update our app afterwards.

**Comment by larsxschneider** [comment]
looks good. should we set our master exactly to the hash of the original repo instead of a merge? (since we don't need any additions anymore)

**Comment by larsxschneider** [comment]
that would be: b5a17d9aaa27f025352545525988019b0c59195a

**Pull request merged**